toolbox/deps_update/main.go

```diff
@@ -146,7 +146,11 @@ func updateDependenciesOf(repo string) error {
 		return err
 	}
 	prTitle := prTitlePrefix + repo
-	return githubClnt.CreatePullRequest(prTitle, prBody, branch, *baseBranch, repo)
+	pr, err := githubClnt.CreatePullRequest(prTitle, prBody, branch, *baseBranch, repo)
+	if err != nil {
+		return err
+	}
+	return githubClnt.MergePullRequestOnceCIGreen(repo, pr)
 }
 
 func main() {
```

Comment:
Can you make sure that error and converted in log.Fatalf. That way the binary will return a non 0 exit code, which will failure the prow job. Currently it fails silently.

Reply:
Done

Reply:
Done